pset-functions.js

```diff
@@ -31,6 +32,38 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+const playerCardScore = 21
+const dealerCardScore = 22
+
+function blackJack (playerCardScore, dealerCardScore) {
+
+  const playerDifference = 21 - playerCardScore
+  const dealerDifference = 21 - dealerCardScore
+
+  if (playerCardScore > 21 && dealerCardScore > 21) {
```

Comment:
@kcycle nice job, good use of `if..else if` to cover all the necessary conditions. Take a look at the solution to view an alternative `if..else`  structure for this problem:  https://github.com/jsd20191008/hw-01-functions-solution/blob/master/pset-functions.js#L33